format/testfiles/test_with.rego.formatted

```diff
@@ -15,6 +15,12 @@ multi_line_with {
 		]
 }
 
+mixed_new_lines_with {
+	true with input.a as "a"
+		with input.b as "b" with input.c as "c"
```

Comment:
I think it would be nice if we put each `with` on its own line, but at least this fixes the bug!

Reply:
#4508 😉

Reply:
Hah! Yeah I suppose so :)